edx_proctoring/api.py

```diff
@@ -635,7 +636,7 @@ def create_exam_attempt(exam_id, user_id, taking_as_proctored=False):
         scheme = 'https' if getattr(settings, 'HTTPS', 'on') == 'on' else 'http'
         lms_host = '{scheme}://{hostname}'.format(scheme=scheme, hostname=settings.SITE_NAME)
 
-        obs_user_id = hashlib.sha1((u'%s%s' % (attempt_code, user_id)).encode('ascii')).hexdigest()
```

Comment:
With the attempt code here, the user id is so obfuscated that it becomes useless. (attempt codes are unique to attempts)

Reply:
course_id is probably too fine-grained.